helm/cluster/files/etc/selinux/config

```diff
@@ -0,0 +1,15 @@
+# This file controls the state of SELinux on the system on boot.
+
+# SELINUX can take one of these three values:
+#       enforcing - SELinux security policy is enforced.
+#       permissive - SELinux prints warnings instead of enforcing.
+#       disabled - No SELinux policy is loaded.
+SELINUX={{ $.Values.global.components.selinux.mode }}
+
+# SELINUXTYPE can take one of these four values:
+#       targeted - Only targeted network daemons are protected.
+#       strict   - Full SELinux protection.
+#       mls      - Full SELinux protection with Multi-Level Security
+#       mcs      - Full SELinux protection with Multi-Category Security
+#                  (mls, but only one sensitivity level)
+SELINUXTYPE=mcs
```

Comment:
> Do we also want to allow to set the type of protection (targeted, strict, mls, mcs) through values?

If `mcs` is the current default, do you see any clear situations where customers would want to change the mode?

IIUC, MCS is actually an extension of MLS, and both supplement `strict`. So in theory, MCS would be the most fully featured option from an enforcement perspective. Is that correct?

Reply:
100%, honestly I don't think customers would want to change it, and I don't think we would want to test it in any other way, so I think we're fine just giving the option to set enforcing/permissive or disable SELinux.

Reply:
That's my impression as well. Then this looks great :+1: